all/pom.xml

```diff
@@ -680,59 +685,7 @@
-                                    <include>io.seata:seata-discovery-etcd3</include>
-                                    <include>io.seata:seata-dubbo</include>
-                                    <include>io.seata:seata-dubbo-alibaba</include>
-                                    <include>io.seata:seata-motan</include>
-                                    <include>io.seata:seata-grpc</include>
-                                    <include>io.seata:seata-http</include>
-                                    <include>io.seata:seata-rm</include>
-                                    <include>io.seata:seata-rm-datasource</include>
-                                    <include>io.seata:seata-sqlparser-core</include>
-                                    <include>io.seata:seata-sqlparser-antlr</include>
-                                    <include>io.seata:seata-sqlparser-druid</include>
-                                    <include>io.seata:seata-sofa-rpc</include>
-                                    <include>io.seata:seata-spring</include>
-                                    <include>io.seata:seata-tcc</include>
-                                    <include>io.seata:seata-tm</include>
-                                    <include>io.seata:seata-serializer-seata</include>
-                                    <include>io.seata:seata-serializer-protobuf</include>
-                                    <include>io.seata:seata-serializer-kryo</include>
-                                    <include>io.seata:seata-serializer-fst</include>
-                                    <include>io.seata:seata-serializer-hessian</include>
-                                    <!-- saga -->
-                                    <include>io.seata:seata-saga-processctrl</include>
-                                    <include>io.seata:seata-saga-statelang</include>
-                                    <include>io.seata:seata-saga-engine</include>
-                                    <include>io.seata:seata-saga-rm</include>
-                                    <include>io.seata:seata-saga-tm</include>
-                                    <include>io.seata:seata-saga-engine-store</include>
-                                    <!--compressor-->
-                                    <include>io.seata:seata-compressor-gzip</include>
-                                    <include>io.seata:seata-compressor-7z</include>
-                                    <include>io.seata:seata-compressor-bzip2</include>
-                                    <include>io.seata:seata-compressor-zip</include>
-                                    <include>io.seata:seata-compressor-lz4</include>
-                                    <include>io.seata:seata-compressor-deflater</include>
-                                    <include>io.seata:seata-compressor-zstd</include>
+                                    <include>io.seata:*</include>
```

Comment:
include seata-server？

Reply:
no, this include only include the <dependencies>.